MusicNotationKit/MusicNotationKit/Tone.swift

```diff
@@ -18,3 +18,13 @@ public struct Tone {
 		self.octave = octave
 	}
 }
+
+extension Tone: CustomDebugStringConvertible {
+	public var debugDescription: String {
+		var result = "\(noteLetter)\(octave.rawValue)"
+		if accidental != nil {
+			result = "[\(result)\(accidental!)]"
+		}
+		return result
+	}
```

Comment:
This seems odd, because you are putting [ ] around it only if it has an accidental? I would assume we would want to either have it around it every time or never. So, I would make that decision and then rewrite using the guidelines stated in the other comment above.

Using guidelines stated above, rewrite as follows:

```swift
return "[\(noteLetter)\(octave.rawValue)\(accidental ?? "")]"
```